atc/atccmd/command.go

```diff
@@ -798,19 +800,17 @@ func (cmd *RunCommand) constructBackendMembers(
 	engine := cmd.constructEngine(
 		pool,
 		workerClient,
-		resourceFetcher,
+		resourceFactory,
 		dbResourceCacheFactory,
 		dbResourceConfigFactory,
 		secretManager,
 		defaultLimits,
 		buildContainerStrategy,
```

Comment:
They *might*, depending on what we decide for Kubernetes runtime v1. It might mean that we keep the abstraction of multiple workers, but one worker has the Kubernetes API slotted in as a client. This would let operators experimentally provision one Kubernetes cluster as a tagged worker, rather than setting up a new foundation for Kubernetes.. we should do some whiteboarding and deeper thinking as a team about this!